src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -290,9 +313,14 @@ std::vector<ov::Tensor> SubgraphBaseTest::calculate_refs() {
     }
 
     const auto& outputs = functionToProcess->outputs();
+    if (outType.size() == 1) {
```

Comment:
remove code under the comment

Reply:
Sorry for the misunderstanding. Fixed in commit https://github.com/openvinotoolkit/openvino/pull/9975/commits/5cab0e488792af6b4787af1bfc82ef2c6175c1b3.